File: Assignment_code_WPT/Day_14_26_09_2025/react-project-2/src/Factorial.js

export default function Factorial(prop){

    let fact = 1
    let num = parseFloat(prop.number)
    for (let a=1;a<=num;a++)
    {
        fact*=a;
    }

    return(<h1>

        {num} FACTORIAL IS {fact}</h1>)
}

File: C++/Day_3-23_08_25/Factorial.cpp
#include<iostream>

using namespace std;

int factorial(int num)
{
    int i,fact=1;
    // cout<<num<<endl;
    //  cout<<fact<<endl;

        for ( i = num; i >= 1; i--)
        {
            fact=fact*i;
            // cout<<fact<<endl;
        }
        // cout<<fact;
        return fact;
        
}
int main()
{
    int num;

    cout<<"Enter the number to calculate factorial :";
    cin>>num;

    cout<<"Factorial of num "<<num<<" is : "<<factorial(num);
}